README.md

```diff
@@ -152,6 +153,53 @@ reformatted connect/routes/api.py
 All done! ✨ 🍰 ✨
 1 file reformatted.
 ```
+## Build multi-arch docker image 
+LinuxForHealth connect runs on amd64, arm64 & s390x platforms.  Follow these instructions to build the multi-arch image that supports all these platforms.  At this time, you need access to all 3 platforms in order to build the multi-arch image.
+
+### Build the amd64 and arm64 image
+Run `docker buildx` to build an image that supports amd64 and arm64.  The s390x image has to be built separately, for now.
+```shell
+docker buildx build --pull --push --platform linux/amd64,linux/arm64 --build-arg APPLICATION_BUILD_CERT_PATH=./local-config/ -t linuxforhealth/connect:0.42.0 .
+```
+
+### Pull, tag and push the amd64 image
+On your amd64 machine:
+```shell
+docker pull linuxforhealth/connect:0.42.0
+docker images
+docker image tag <image_id> linuxforhealth/connect:0.42.0-amd64
+docker push linuxforhealth/connect:0.42.0-amd64
+```
+
+### Pull, tag and push the arm64 image
+On your arm64 device:
+```shell
+docker pull linuxforhealth/connect:0.42.0
+docker images
+docker image tag <image_id> linuxforhealth/connect:0.42.0-arm64
+docker push linuxforhealth/connect:0.42.0-arm64
+```
+
+### Build, tag and push the s390x image
+On your s390x machine, build the s390x connect image using these [instructions](./platforms/s390x/README.md), then tag and push the image:
+```shell
+docker images
+docker image tag <image_id> linuxforhealth/connect:0.42.0-s390x
+docker push linuxforhealth/connect:0.42.0-s390x
+```
+
+### Create the multi-arch image
+Use `docker manifest` to create the multi-arch image for all 3 platforms and push it:
```

Comment:
<img width="731" alt="Screen Shot 2021-08-04 at 7 39 22 AM" src="https://user-images.githubusercontent.com/39178401/128174617-55b2f4fb-5c63-49ae-92ce-6bba2fb022a2.png">

Do we need to leave the platform specific images on docker hub, or can clients just download the "connect:0.42.0" tag? If we drill into the "connect:0.42.0" tag on docker hub we can select the different architectures.

Reply:
The only reason to leave the -arch images is to easily recreate the multi-arch image if the 0.42.0-tagged image gets overwritten with a single-arch image, say the amd64 version.